plugins/node.d/icinga_hosts

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+
+##%# family=auto
```

Comment:
Please add a perldoc header here (see for example the `plugins/node.d.linux/if_` plugin).